packages/meteor/test_environment.js

```diff
@@ -10,8 +12,29 @@ var testDriverPackageName = TEST_METADATA.driverPackage;
 
 // Note that if we are in test-packages mode neither of these will be set,
 // but we will have a test driver package
+/** 
+ *@memberof Meteor
+ * @summary Boolean variable.  True if running in test environment.
```

Comment:
Sometimes this is false in test environments: https://github.com/meteor/meteor/blob/77df795928df55d1e114cb4d2a9143e433a9e773/tools/cli/commands.js#L1995-L1996

Reply:
What would be the proper description then? ` True if running in test environment, but can be set to false if test are being run in full-app mode`?

Reply:
Maybe something like `True when running tests, except for when running in full-app mode`. Or maybe `True when running unit tests (false if running tests in full app mode)`.